public/app/core/services/echo/backends/analytics/ApplicationInsightsBackend.ts

```diff
@@ -0,0 +1,46 @@
+import $ from 'jquery';
+import { EchoBackend, EchoEventType, isInteractionEvent, isPageviewEvent, PageviewEchoEvent } from '@grafana/runtime';
+
+export interface ApplicationInsightsBackendOptions {
+  connectionString: string;
+}
+
+export class ApplicationInsightsBackend implements EchoBackend<PageviewEchoEvent, ApplicationInsightsBackendOptions> {
+  supportedEvents = [EchoEventType.Pageview];
+
+  constructor(public options: ApplicationInsightsBackendOptions) {
+    $.ajax({
+      url: 'https://js.monitor.azure.com/scripts/b/ai.2.min.js',
+      dataType: 'script',
+      cache: true,
+    }).done(function () {
+      var applicationInsightsOpts = {
+        config: {
+          connectionString: options.connectionString,
```

Comment:
Is it possible to add another config option for endpointUrl to be set from the Grafana config? Different Azure clouds need to be configured with different data collection endpoints. More info here: https://docs.microsoft.com/en-us/azure/azure-monitor/app/custom-endpoints?tabs=js

Reply:
Sure, I just pushed some changes. Please take another look